docker/production/entrypoint-app.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+cd /usr/src/app
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)